### PR TITLE
[PWX-33947] Write to expanded size of the pool after resize of the pool

### PR DESCRIPTION
**Pull request by rkinhalkar-px**
Automation test for Write to expanded size of the pool after resize of the pool

PWX-33947

Test Pass Jobs :- 
https://jenkins.pwx.dev.purestorage.com/user/ragsingh@purestorage.com/my-views/view/all/job/Users/job/Raghvendra/job/raghav_pxdv-7-dev/88/
https://jenkins.pwx.dev.purestorage.com/user/ragsingh@purestorage.com/my-views/view/all/job/Users/job/Raghvendra/job/raghav_pxdv-7-dev/89/

### CODE REVIEW

File: tests/basic/pool_function_test.go [hunk 1]
```diff
@@ -1423,3 +1425,115 @@ var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
...
+		}
+		selectedPool, err := GetStoragePoolByUUID(offlinePoolUUID)
+		log.FailOnError(err, "error getting pool with UUID [%s]", offlinePoolUUID)
+
+		var expandedExpectedPoolSize uint64
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			expandedExpectedPoolSize = (selectedPool.TotalSize / units.GiB) * 2
+
+			log.FailOnError(err, "Failed to check if Journal enabled")
```
Comment (vinayakshnd) [inline tests/basic/pool_function_test.go:1507]: Is this line required here?

File: tests/basic/pool_function_test.go [hunk 2]
```diff
@@ -1423,3 +1425,115 @@ var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
...
+		isjournal, err := IsJournalEnabled()
+		log.FailOnError(err, "is journal enabled check failed")
+
+		err = adjustReplPools(*selectedNode, secondReplNode, isjournal)
+		log.FailOnError(err, "Error setting pools for clean volumes")
+
+		Inst().AppList = []string{"fio-fastpath"}
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("sfullrz-%d", i))...)
```
Comment (vinayakshnd) [inline tests/basic/pool_function_test.go:1482]: Validate deployed apps here.

**Review by vinayakshnd — APPROVED** [review]
Provided couple of minor comments. Address them before merging.

File: tests/basic/pool_function_test.go (hunk 1 above)
Comment (rkinhalkar-px) [reply tests/basic/pool_function_test.go:1507]: removed

File: tests/basic/pool_function_test.go (hunk 2 above)
Comment (rkinhalkar-px) [reply tests/basic/pool_function_test.go:1482]: done

**Pull request merged**